spec/components/show/item/overview_component_spec.rb

```diff
@@ -7,20 +7,7 @@
   let(:presenter) { instance_double(ArgoShowPresenter, document: doc, cocina: cocina, change_set: change_set, state_service: state_service) }
   let(:change_set) { ItemChangeSet.new(cocina) }
   let(:cocina) do
-    Cocina::Models::DRO.new(externalIdentifier: 'druid:bc234fg5678',
-                            type: Cocina::Models::ObjectType.document,
-                            label: 'my dro',
-                            version: 1,
-                            description: {
-                              title: [{ value: 'my dro' }],
-                              purl: 'https://purl.stanford.edu/bc234fg5678'
-                            },
-                            access: {},
-                            identification: {},
-                            structural: {},
-                            administrative: {
-                              hasAdminPolicy: 'druid:hv992ry2431'
-                            })
+    build_for_repository(:item)
```

Comment:
What is the distinction between `build_for_repository` and `create_for_repository`? Also, the "_for_repository" feels a bit odd. "_cocina" instead?

Reply:
We've had `create_for_repository` in this repo for 2 years. I think it's pretty good. It was copied from Hyrax: https://github.com/samvera/hyrax/blob/573f1ce2f9eea19b32bf9905b1725db63d5be730/spec/support/create_for_repository.rb